Public/Src/Utilities/Native/IO/IFileSystem.cs

```diff
@@ -243,12 +243,12 @@ public interface IFileSystem
         FileFlagsAndAttributes GetFileFlagsAndAttributesForPossibleReparsePoint(string expandedPath);
 
         /// <summary>
-        /// Thin wrapper for native GetFileAttributesW that throws an exception on failure
+        /// Thin wrapper for native GetFileAttributesW that throws an exception on failure by default

```

Comment:
what does it do when an error happens but `throwOnFailure` is `false`?

Reply:
In both implementation it returns `FileAttributes.Normal` - the Windows implementation does that since the beginning.